evals/elsuite/basic/match.py

```diff
@@ -36,18 +35,16 @@ def eval_sample(self, sample: Any, *_):
                 prompt += s["sample"]
             prompt += sample["input"][-1:]
 
-        # TODO(hwc): is there a case where we want to use `result` other than "choices"?
-        result = self._completion_fn(
+        result = self.completion_fn(
```

Comment:
Current setup leaves it up to the eval implementation to decide what to do with the multiple completion_fns. See the changes in ModelBasedClassify for an example of using multiple completion_fns (it uses self.completion_fns instead of self.completion_fn)

Reply:
Oh interesting, I took it to mean that if you have multiple completion_fns, you will run the eval with each completion_fun,

so if we do `oaieval gpt-3.5-turbo,text-davinci-003 test-match` it would be equivalent to doing  `oaieval gpt-3.5-turbo test-match` and  `oaieval text-davinci-003 test-match` individually. However it looks like it's used differently in classify.py